src/components/WidgetHugger.vue

```diff
@@ -16,8 +16,17 @@
   />
   <div ref="outerWidgetRef" class="outerWidget">
     <div
+      v-if="widgetHasOwnContextMenu[widget.component as WidgetType]"
+      class="innerWidget"
+      :class="{ 'overflow-hidden': hideOverflow }"
+      :style="{ opacity: widget.options.opacity ?? 1 }"
+    >
+      <slot></slot>
+    </div>
+    <div
+      v-else
       ref="innerWidgetRef"
-      v-contextmenu="handleContextMenu"
+      :v-contextmenu="widget.options.blockGenericContextMenu ? null : handleContextMenu"
```

Comment:
Everything working fine here!
There's just this leftover from the last review.
Besides that it's ready to approve from my side.